question: path alias in both tsconfig?

I hit an issue where i get runtime errors without this path alias https://github.com/dan5py/turborepo-shadcn-ui/blob/main/packages/typescript-config/nextjs.json#L13 like 
```
web:dev:  ⨯ ../../packages/ui/src/components/ui/button.tsx:4:0
web:dev: Module not found: Can't resolve '@/utils/cn'.
``` 
could you please elaborate why having only this one is not enough https://github.com/dan5py/turborepo-shadcn-ui/blob/main/packages/ui/tsconfig.json#L6 . does it mean if i want to use alias like @/* in both package/ui and apps/docs they will be conflicting and I need separate alias for ui package? 
am I better just go without aliases?

--- a/apps/docs/tsconfig.json
+++ b/apps/docs/tsconfig.json
@@ -5,7 +5,10 @@
       {
         "name": "next"
       }
-    ]
+    ],
+    "paths": {
+      "~/*": ["./*"]
+    }
   },
   "include": [
     "next-env.d.ts",

--- a/packages/typescript-config/nextjs.json
+++ b/packages/typescript-config/nextjs.json
@@ -8,9 +8,6 @@
     "moduleResolution": "Bundler",
     "allowJs": true,
     "jsx": "preserve",
-    "noEmit": true,
-    "paths": {
-      "@ui/*": ["../../packages/ui/src/*"]
-    }
+    "noEmit": true
   }
 }

--- a/packages/ui/components.json
+++ b/packages/ui/components.json
@@ -11,7 +11,7 @@
     "prefix": ""
   },
   "aliases": {
-    "components": "@ui/components",
-    "utils": "@ui/lib/utils"
+    "components": "@repo/ui/components",
+    "utils": "@repo/ui/lib/utils"
   }
 }

--- a/packages/ui/package.json
+++ b/packages/ui/package.json
@@ -33,6 +33,9 @@
     "./postcss.config": "./postcss.config.mjs",
     "./tailwind.config": "./tailwind.config.ts",
     "./lib/*": "./src/lib/*.ts",
-    "./components/*": "./src/components/*.tsx"
+    "./components/*": [
+      "./src/components/*.tsx",
+      "./src/components/*.ts"
+    ]
   }
 }

--- a/packages/ui/src/components/ui/button.tsx
+++ b/packages/ui/src/components/ui/button.tsx
@@ -2,7 +2,7 @@ import * as React from "react";
 import { Slot } from "@radix-ui/react-slot";
 import { cva, type VariantProps } from "class-variance-authority";
 
-import { cn } from "@ui/lib/utils";
+import { cn } from "@repo/ui/lib/utils";
 
 const buttonVariants = cva(
   "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50",

--- a/packages/ui/tsconfig.json
+++ b/packages/ui/tsconfig.json
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@ui/*": ["./src/*"]
+      "@repo/ui/*": ["./src/*"]
     }
   },
   "include": ["src"],

